improve deep create with troubleshooting (#77)
SAP/cloud-sdk: docs/js/features/odata/common/operations/deep-create.mdx

BusinessPartner.requestBuilder().create(businessPartner).execute(myDestination);
```

:::note Troubleshooting
When you try the example code above for testing the `deep creat` feature, you might see some errors like `"operation module BUA_CHECK_ADDRESS_VALIDITY_ALL; a check table is missing"`.

> Typically, it can happen if you are using some new systems with default configuration.

Reviewer: ```suggestion
Typically, it can happen if you are using a new system with a default configuration.
```

> To fix it, the `address usage` needs to be provided.
> Please find the example code below.

Reviewer: ```suggestion
You need to configure an `address usage` field as shown in the example below to fix it.
```

Reviewer: Is it Ok to call `address usage` a field? Or is it a parameter? Or is it a configuration?
